### PR TITLE
Reduce verbosity of ES update job, refs #13085

### PR DESCRIPTION
**Pull request by djjuhasz**
Reduce status updates to reporting after every 100 description updates
to avoid overflowing jobs.output column width.

### CODE REVIEW

File: lib/job/arUpdateEsIoDocumentsJob.class.php [hunk 1]
```diff
@@ -56,36 +56,44 @@ public function runJob($parameters)
...
-      $this->info($message);
+      // Status update every 100 descriptions
+      if (0 == $count % 100)
+      {
+        $this->info($message);
+      }
     }
 
+    // Final status update
+    $this->info($message);
```
Comment (sbreker) [inline lib/job/arUpdateEsIoDocumentsJob.class.php]: This message will be printed twice if the number of objects being indexed is a multiple of 100.

**Review by sbreker — COMMENTED** [review]
Looks great! One nitpick regarding that last iteration print statement.

File: lib/job/arUpdateEsIoDocumentsJob.class.php (hunk 1 above)
Comment (djjuhasz) [reply lib/job/arUpdateEsIoDocumentsJob.class.php]: Thanks @sbreker, I've fixed this.

**Review by sbreker — APPROVED** [review]
Looks good @djjuhasz!

**Comment by djjuhasz** [comment]
Merged to qa/2.6.x with 2976888352adf9b2661e4207e2a038c8fed8af34

**Pull request closed without merging**